### ISSUE TITLE
Exclude imports from graphql-codegen-typescript-react-apollo-template

### ISSUE THREAD

**Issue by moonray** [dotansimha/graphql-code-generator#744]
**Is your feature request related to a problem? Please describe.**
tslint complains about unused imports:
```
'ReactApollo' is declared but its value is never read.
'React' is declared but its value is never read.
'gql' is declared but its value is never read.
```

**Describe the solution you'd like**
Have a way through config to exclude these imports.

**Describe alternatives you've considered**
using `sed` or something similar to hack the template's output

**Additional context**
code in question:
````typescript
import * as ReactApollo from 'react-apollo';
import * as React from 'react';

import gql from 'graphql-tag';
````

**Comment by ardatan** [comment]
Could you share whole generated file?

**Comment by kamilkisiela** [comment]
@ardatan maybe there's no operations?

**Comment by dotansimha** [comment]
@moonray available in 0.13.0.
